docs/references/go/overview.mdx

```diff
@@ -5,32 +5,113 @@ description: Learn how to integrate Go into your Clerk application.
 
 # Go
 
-Clerk's Go SDK is a thin wrapper over the Clerk Backend API. Add the following import statement:
+Clerk's [Go SDK](https://github.com/clerk/clerk-sdk-go) is a wrapper over the [Clerk Backend API](https://clerk.com/docs/reference/backend-api).
+
+## Installation
+
+If you are using Go Modules and have a `go.mod` file in your project's root, you can import `clerk-sdk-go` directly.
 
 ```go
```

Comment:
Should this have the `go.mod` file path?

Reply:
No, this import can (and will) be defined in every file you need to access the `clerk` package.